### ISSUE TITLE
Fix down carpet edging strips in the main room

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#1058]
In #1029 @magman2112 and I fastened down the edging strip between the carpet and varnished floor by the UX lab.  We should screw down the remaining strips: between the banks of desks and by the recess.

@magman2112 Did you have some plans for better covering of the power and network cables running across between the banks of desks?

**Comment by magman2112** [comment]
Yes, I plan to make some aluminium covers to protect these cables, in addition to the existing carpet edging strips. I will try to remember to bring my sheet metal folder in on Thursday and see if someone wants to assist me with this task.

**Comment by magman2112** [comment]
The carpet edging strip by the 3D printers has now been screwed down. I will look into making the cable protectors in the next few weeks so that we can close this issue.

**Comment by johnmckerrell** [comment]
I think this had been resolved now?

**Comment by amcewen** [comment]
We don't have the cable protectors, and the gaffer tape that has been in use instead is getting to a point where it could do with being replaced.

We'll need to lift it to get at the extension lead that runs under it for PAT testing too, so replacing it after the PAT test seems a sensible approach.

I wonder if https://www.screwfix.com/p/extra-wide-carpet-cover-door-strip-aluminium-0-9m-x-61mm/81951 would be wide enough to modify (maybe bend it to a better profile, and fix it along the edges rather than in the centre) to work as a more suitable replacement?
